reactioncommerce/api-plugin-tags
# feat: move source code and enable semantic-release

This plugin replaces the internal `tags` service.

This PR:
- copies the source code from `reaction`
- updates the licence to GPL-3
- enables semantic-release

Once this PR is merged and this plugin is published to `npm`, there will be a related PR in `reaction` to swap out this plugin for the existing version.

## Testing

- see that the plugin is loaded upon startup in Reaction (use the `systemInformation` query)
- use the `tags` queries and mutations through the GraphQL API and verify they are working

Issue https://github.com/reactioncommerce/reaction/issues/6181

Signed-off-by: Maria Paktiti <maria.paktiti@gmail.com>
